src/cluster/Backend.h

```diff
@@ -0,0 +1,457 @@
+// See the file "COPYING" in the main distribution directory for copyright.
+
+// The interface for cluster backends and remote events.
+
+#pragma once
+
+#include <memory>
+#include <mutex>
+#include <optional>
+#include <string_view>
+#include <variant>
+
+#include "zeek/EventHandler.h"
+#include "zeek/Flare.h"
+#include "zeek/IntrusivePtr.h"
+#include "zeek/Span.h"
+#include "zeek/cluster/Serializer.h"
+#include "zeek/iosource/IOSource.h"
+#include "zeek/logging/Types.h"
+
+namespace zeek {
+
+class Val;
+class FuncVal;
+
+using ValPtr = IntrusivePtr<Val>;
```

Comment:
I think this line can go?

Reply:
It's used below for ArgsSpan - I'll reorder a bit.